backend/functions/orders-sync/orders-sync.function.ts

```diff
@@ -1,23 +1,33 @@
 // TODO: The original context was InvocationContext
-import { Context } from '@azure/functions';
-import ContextCreator from '../lib/adapters/utils/application-context-creator';
+import { app, InvocationContext, Timer } from '@azure/functions';
+import ContextCreator from '../azure/application-context-creator';
 import { initializeApplicationInsights } from '../azure/app-insights';
 import { OrdersController } from '../lib/controllers/orders/orders.controller';
 
 import * as dotenv from 'dotenv';
+import { toAzureError } from '../azure/functions';
 
 // TODO: We need to look into upgrading this to use v4 of Azure Functions
```

Comment:
`// TODO: We need to look into upgrading this to use v4 of Azure Functions
Comment` Is this still something we need to look into? Is this talking about the use of `dotenv` or the function in general?

Reply:
I don't think they have anything to do with one another. I'd be happy to get rid of that dependency if we don't need it though.

Reply:
Punting for now.